lib/core-generators/new/templates/views/pages/legal/privacy.ejs

```diff
@@ -38,17 +38,36 @@
 
+    <h3>How long we store information</h3>
+    <p>NEW_APP_COMPANY_NAME will store your data as long as needed to provide you with our services and to operate our organization. If you ask NEW_APP_COMPANY_NAME to delete specific personal information, we'll honor your request unless deleting that information prevents us from carrying out necessary functions, like calculating taxes or conducting required audits.</p>
+
     <h3>Security</h3>
-    <p>We have security measures in place to help protect against the unauthorized access of personally identifiable information under our control. We utilize both online and offline security methods, including firewalls, passwords and restricted physical access to the places where your information is stored to help protect personally identifiable information. Our staff is trained to comply with our security procedures, and our security procedures are regularly reviewed and revised as we deem necessary.</p>
+    <p>We have security measures in place to help protect against the unauthorized access to personally identifiable information under our control. We utilize both online and offline security methods, including firewalls, passwords and restricted physical access to the places where your information is stored to help protect personally identifiable information. Our staff is trained to comply with our security procedures, and our security procedures are regularly reviewed and revised as we deem necessary.</p>
 
     <h3>How you can access and update your information</h3>
     <p>Our goal is for any personal information in our possession to be as accurate, current and complete as necessary for the purposes for which we use that information. If you want to review, verify or correct your personal information, please contact us using the contact information set out below. We reserve the right not to change any personal information and instead append any alternative information the individual concerned believes is appropriate. To some extent, we rely on individuals to update their own personal information. Unless an individual advises us of these important changes, we may have no way of knowing about them.</p>
+
+    <h3>Your legal rights</h3>
     <p>The State of California enacted the Shine the Light law (CA Civil Code Sec 198.83) that permits users who are California residents to request certain information regarding the disclosure of personal information during the past year for marketing purposes. To make such a request, you can <a href="/contact">contact us</a>.</p>
+    <p>If you are in the European Union, you have rights under the General Data Protection Regulation in relation to your personal data:</p>
+    <ul>
+      <li><strong>The right to access</strong> - You have the right to request NEW_APP_COMPANY_NAME for copies of your personal data. We may charge you a small fee for this service.</li>
```

Comment:
How about instead of this, we go with:

> **Access** - You are welcome to request copies of your personal data from us.  We may charge you a small fee for this service.

Seem ok?  If you like, then let's follow the same approach for the others, making the language as friendly as possible within the boundaries of the legal framework we're stuck with